packages/markdown-slate/src/SlateTransformer.js

```diff
@@ -51,6 +51,20 @@ class SlateTransformer {
             object: 'value',
             document: parameters.result
         };
+        const paragraphSpaceNodeJSON = {
+            object: 'block',
+            type: 'paragraph',
+            data: {
+            },
+            nodes: [
+                {
+                    object: 'text',
+                    text: '',
+                    marks: []
+                }
+            ]
+        };
+        result.document.nodes.push(paragraphSpaceNodeJSON);
```

Comment:
If you only added this when the last block was a Clause then you wouldn't need to change all the tests... ;-)